### ISSUE TITLE
Visual Studio version and project compatibility.

### ISSUE THREAD

**Issue by drgarciafuentes** [marcingminski/sql-performance-monitor#444]
**The issue**
Marcin,
I've been following your project for quite some time but even after reading the documentation and going through the source code and project files I can't for the love of god figure out which version of Visual Studio you have been using.
Your documentation does not specify which version you used on each release. I know that you have used 'express edition' now days 'community edition' but some part like the SSIS projects brake compatibility so easily that it is hard to follow with MS.

I started looking at your project since version 3.x 

Is it possible for the repo and docs to have this information available under the 'deploy from code' section?

Thanks in advance.

**Comment by marcingminski** [comment]
The latest will be fine (2019 as there no SSIS add on for 2022 yet)

But yeah, good idea about mentioning which version was used for development with each release. Thanks !

**Comment by drgarciafuentes** [comment]
Yes, it is really helpful to be able to follow which version of the project matches which release of VS.
Not sure if you would ever look into making the remote server data available through an API but it would be a supper handy feature.

Thank you for taking a look at this issue and responding so fast.

**Comment by marcingminski** [comment]
> Not sure if you would ever look into making the remote server data available through an API but it would be a supper handy feature.

This is something that is currently a work in progress as a paid (although very low cost) service. Let me know if you would like to know more.

**Comment by marcingminski** [comment]
> Yes, it is really helpful to be able to follow which version of the project matches which release of VS. 
> 
> Thank you for taking a look at this issue and responding so fast.

Hi, where would you want to have this documented?

**Comment by marcingminski** [comment]
Hi, I just had a look at `SqlWatch.Monitor.sln` and it seem to contain all the relevant info:
```
Microsoft Visual Studio Solution File, Format Version 12.00
# Visual Studio Version 16
VisualStudioVersion = 16.0.30309.148
MinimumVisualStudioVersion = 10.0.40219.1
```
I am going to update documentation to make users aware they can check the version in this file.
